### DISCUSSION TITLE
Can't access Tactical RMM from local network

### DISCUSSION THREAD

**Discussion by Albinana** [amidaware/tacticalrmm#1696] [General]
**Server Info (please complete the following information):**
 - OS: Debian 12
 - Browser: Chrome
 - RMM Version (as shown in top left of web UI): lastest

**Installation Method:**
  - [x ] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version (as shown in the 'Summary' tab of the agent from web UI): lastest
- Agent OS: [e.g. Win 10 v2004, Server 2012 R2]

**Describe the bug**
I can acces TacticalRMM from outside the network, but not on the local network where the server is. 
+ have problems of cert inside my local network but not outside

Probably a problem of port

Pls send help

**Comment by silversword411** [comment]
You probably have a DNS or other routing problem with your installation. Paid support is available here: https://docs.tacticalrmm.com/support/#commercial-support
